packages/block-library/src/site-title/edit/index.js

```diff
@@ -87,6 +96,28 @@ export default function SiteTitleEdit( {
 					} }
 				/>
 			</BlockControls>
+			<InspectorControls>
+				<PanelBody title={ __( 'Link settings' ) }>
+					<ToggleControl
+						label={ __( 'Make title link to home' ) }
```

Comment:
For the post title the text is: `Make title a link`, it assumes the user knows its a link to the post.
Should this text be the same, or `Make title a link to home` ?
```suggestion
						label={ __( 'Make title a link to home' ) }
```

Reply:
Could just `Link to home` work?